pkg/logger/logger_test.go

```diff
@@ -120,3 +121,63 @@ func TestDebugPayload(t *testing.T) {
 		})
 	}
 }
+
+func TestGetLogLevelByNameWithSuffix(t *testing.T) {
```

Comment:
Didn't catch this before, but can we come up with a better name? We're not testing the ability to get the log level by name. We're testing that a configured suffix appears in the log message.